packages/massa-web3/src/experimental/basicElements/mas.ts

```diff
@@ -19,12 +19,8 @@ export type Mas = bigint
  *
  * @throws An error if the value is not a safe integer.
  */
-export function fromMas(value: number): Mas {
-  if (!Number.isSafeInteger(value)) {
-    throw new Error(ERROR_NOT_SAFE_INTEGER)
-  }
-
-  return BigInt(value * POWER_TEN ** NB_DECIMALS)
+export function fromMas(value: U64): Mas {
+  return fromNumber(value * BigInt(POWER_TEN) ** BigInt(NB_DECIMALS))
```

Comment:
Naming the variable TEN should be enough so we can move it to the no magic number file

Reply:
Even if TEN and POWER_TEN have the same underlying value, having TEN doesn't explain the reason behind our actions. 
Here, we want to convert the integer value to the expected numerical class in base 10. That’s why we use a power of ten for this conversion.
Similarly, we use variables like milli, micro, and nano to make the code more readable. This approach clarifies the purpose and improves the understanding of the transformations being applied.